validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/AggregationDutyTest.java

```diff
@@ -290,6 +293,83 @@ public void shouldProduceSingleAggregateAndProofWhenMultipleValidatorsAggregateS
         .record(any(), any(AggregationDuty.class), eq(ValidatorDutyMetricsSteps.SIGN));
   }
 
+  @TestTemplate
+  public void
+      shouldProduceAllAggregateAndProofsWhenValidatorsAggregateSameCommitteeAndUsingUngroupedAggregators() {
```

Comment:
this is probably at the point where a comment would be better than a ridiculous name...